server/app/models/DisplayMode.java

```diff
@@ -12,17 +12,44 @@
 public enum DisplayMode {
   // The following modes are mutually exclusive.
   // The program should be fully visible.
-  PUBLIC,
+  PUBLIC {
+    @Override
+    public String getVisibilityStateString() {
+      return "Public";
```

Comment:
Should these be localized strings?

Reply:
Good question- we do not have localization for Admin pages. May be a future work item.